src/views/LoginSuccess.vue

```diff
@@ -112,10 +117,24 @@ export default class LoginSuccess extends Vue {
         this.done();
     }
 
-    private done() {
+    private async done() {
         if (!this.walletInfos.length) throw new Error('WalletInfo not ready.');
 
-        this.result = this.walletInfos.map((walletInfo) => ({
+        if (BrowserDetection.isIOS() || BrowserDetection.isSafari()) {
+            const infoEntries = this.walletInfos.map((walletInfo) => walletInfo.toObject());
+            if (!await CookieHelper.canFitNewWallets(infoEntries)) {
```

Comment:
Seems mostly like a duplicate of the `NotEnoughCookieSpace` component. However there it is classified as an error, here it is a `Loader.State.Warning`. I would say it should rather be the other way around. In case you already added an Account that cannot be stored it should be an error, while attempting to add one should result in a Warning if there is not enough space to do so.

Reply:
Made it an error.